pglogical/hooks.py

```diff
@@ -33,7 +41,8 @@ def post_load():
 
     def execute(self, query, params=None, log_exceptions=None):
         """Wrap DDL in pglogical.replicate_ddl_command"""
-        if (query[:6] == "CREATE" or query[:5] == "ALTER" or query[:4] == "DROP") and (
+        parsed_queries = [x.get_type() for x in sqlparse.parse(query)]
+        if any(q in ["CREATE", "ALTER", "DROP"] for q in parsed_queries) and (
             # don't replicate constraints, triggers, indexes
             "CONSTRAINT" not in query
```

Comment:
we should be doing this part with sqlparse then too, right?

Reply:
I left this part as is in order not to complicate/slow it down, but I agree as we could be skipping things otherwise. Your suggestion below would already manage it in a better way.